### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by PrantoshB**
## Hello there 👋🏽
---

In this PR, I have done the followings:
- Modify `schema.sql`
- Add data to `species` column
- Write queries

### Screenshots:

- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
mon

- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
poke

- Delete all records in table and roll back the transaction
delete

- Delete all animals born after Jan 1st, 2022
dob

- Create a savepoint for the transaction and update all animals' weight to be their weight multiplied by -1.
savepoint and weight update

- Rollback to the savepoint
rollback to sp1

- Update all animals' weights that are negative to be their weight multiplied by -1.
update negative weight

- How many animals are there?
total animals

- How many animals have never tried to escape?
count escape 0

- What is the average weight of animals?
average weight

- Who escapes the most, neutered or not neutered animals?
neutered escape

- What is the minimum and maximum weight of each type of animal?
max min weight

- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
escape attempts 1990-2000

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE neutered = true;
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE name != 'Gabumon';
+
```
Comment (Owusu-Desmond) [inline queries.sql:9]: - [ ] Please form the project requirement you have to write queries to answer the following questions and add them to your `queries.sql` :pray: 
  - How many animals are there?
  - How many animals have never tried to escape?
  - What is the average weight of animals?
  - Who escapes the most, neutered or not neutered animals?
  - What is the minimum and maximum weight of each type of animal?
  - What is the average number of escape attempts per animal type of those born between 1990 and 2000?
  
**I can see you performed these queries in your PR but you did not add the queries to this file.** :blush:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @PrantoshB , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (PrantoshB) [reply queries.sql:9]: Thanks @Owusu-Desmond  for pointing this out. I actually added those, but as I forgot to merge the previous PR, it created this conflict. 🤓

**Review by uwadonat — APPROVED** [review]
## Hi @PrantoshB,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Correct workflow was used ✔️
- Well organized and clean project ✔️
thanks-awesome

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**